### DISCUSSION TITLE
What does `BST mean?

### DISCUSSION THREAD

**Discussion by darshanbaral** [opendcs/opendcs#257] [Q&A]
I see <code>`BST</code> in a lot of pseudo-binary dcp messages. Does anyone know what that indicates?

**Comment by MikeNeilson** [comment]
Can you post the whole message. If we take the nesdis we can get a better idea about what the data is.

`BST is likely a value that's just generally static so you often see the same text. A battery voltage on a AC power charger would likely give a constant value, or a tipping bucket. Unfortunately with pseudo-binary just looking at the text doesn't really tell us much.

Or at least me. `BST could have some specific meaning on a given manufacture's DCP.

**Comment by darshanbaral** [comment]
The <code>`BST</code> will be right after the [37 byte header](https://eddn.usgs.gov/dcpformat.html) at the beginning of the message body. At this point, I think it might just be something specific to the equipment.
